SentryPlugin.php

```diff
@@ -112,5 +129,50 @@ public function init()
         craft()->onError = function ($event) use ($client) {
             $client->captureMessage($event->message);
         };
+
+        return $this;
+    }
+
+
+    /**
+     * Includes the JS for front-end sentry.
+     * @return $this
+     */
+    protected function configureFrontendSentry()
+    {
+        // Get plugin settings
+        $settings = $this->getSettings();
+
+        if (!$this->isWebRequest()) {
+            return $this;
+        }
+
+        if (!$settings->reportJsErrors) {
+            return $this;
+        }
+
+        craft()->templates->includeJsFile('https://cdn.ravenjs.com/1.1.22/jquery,native/raven.min.js');
```

Comment:
I don't actually think it is including jquery. Yeah it looks that way
though until I checked what code was actually included and it was not.​

On Fri, Oct 23, 2015 at 2:26 PM, Lawrence <notifications@github.com> wrote:

> In SentryPlugin.php
> <https://github.com/bigcommerce-labs/craft-sentry/pull/1#discussion_r42904676>
> :
>
> > +     * @return $this
> > +     */
> > +    protected function configureFrontendSentry()
> > +    {
> > +        // Get plugin settings
> > +        $settings = $this->getSettings();
> > +
> > +        if (!$this->isWebRequest()) {
> > +            return $this;
> > +        }
> > +
> > +        if (!$settings->reportJsErrors) {
> > +            return $this;
> > +        }
> > +
> > +        craft()->templates->includeJsFile('https://cdn.ravenjs.com/1.1.22/jquery,native/raven.min.js');
>
> @benjolabs <https://github.com/benjolabs> : is including what looks like
> another version of jQuery going to mess with any of our FE stuff?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bigcommerce-labs/craft-sentry/pull/1/files#r42904676>.
>